libs/shared/feature/schema-nuv/src/lib/components/schema-select/formio/formio-schema-select.component.ts

```diff
@@ -60,8 +64,30 @@ export class FormioSchemaSelectComponent extends FormioBaseCustomComponent<ISche
   private _selectSchema(value: string) {
     const selectedSchema = this.schemas.filter(schema => schema.key === value);
     if (selectedSchema && selectedSchema.length > 0) {
-      this.updateValue(selectedSchema[0]);
-      this._changeDetectorRef.markForCheck();
+      this.triggerUpdateValue(selectedSchema[0]);
+    }
+  }
+
+  public loadSchemaWithChildren(selectedOption: INuverialSelectOption): void {
+    if (!this.gotChildrenMap.get(selectedOption.key)) {
+      this._workApiRoutesService
+        .getSchemaTree$(selectedOption.key)
+        .pipe(
+          tap((schema: ISchemaDefinition) => {
+            this.gotChildrenMap.set(schema.key, true);
+            const index: number = this.schemas.findIndex(item => item.key === schema.key);
+            if (index !== -1) {
+              this.schemas[index] = schema;
+              this.triggerUpdateValue(schema);
+            }
+          }),
+        )
```

Comment:
Lets add take to confirm this completes

```suggestion
        .pipe(
          tap((schema: ISchemaDefinition) => {
            this.gotChildrenMap.set(schema.key, true);
            const index: number = this.schemas.findIndex(item => item.key === schema.key);
            if (index !== -1) {
              this.schemas[index] = schema;
              this.triggerUpdateValue(schema);
            }
          }),
          take(1),
        )
```

Reply:
done